projects/ngx-d3timeline/src/lib/axis/time-axis/time-axis-tick-renderer.ts

```diff
@@ -1,8 +1,16 @@
 import { TimeScale } from '../../scale-types';
 import { TickRenderer } from '../tick-renderer';
+import { Orientation } from '../../orientation';
+import { TimelineView } from '../../view/timeline-view';
+import { OptionsService } from '../../options.service';
 
 export class TimeAxisTickRenderer implements TickRenderer {
-  constructor(private scale: TimeScale) {}
+  constructor(
+    private scale: TimeScale,
```

Comment:
only optionsService should be part of the constructor, the other arguments should be passed into the functions. This will allow the renderer to be injected into the axis service

Reply:
As above will address in #107